neqo-transport/src/crypto.rs

```diff
@@ -1493,13 +1494,13 @@ impl CryptoStreams {
         tokens: &mut Vec<RecoveryToken>,
         stats: &mut FrameStats,
     ) {
-        let cs = self.get_mut(space).unwrap();
-        if let Some((offset, data)) = cs.tx.next_bytes() {
+        fn write_chunk(chunk: (u64, &[u8]), builder: &mut PacketBuilder) -> Option<(u64, usize)> {
```

Comment:
I think that you want to make offset and data separate arguments to this.

The calls below all have separate values.

```suggestion
        fn write_chunk(offset: u64, data: &[u8], builder: &mut PacketBuilder) -> Option<(u64, usize)> {
```

(Eat that GitHub UI, I can type a suggestion even if you don't show the button!)